File: src/main/java/org/antop/java/chapter2/ImportFunction.java
package org.antop.java.chapter2;

import org.apache.commons.lang3.StringUtils;

public class ImportFunction {
    /**
     * 전문에서 트랜잭션 코드 추출
     *
     * @param message 전문
     * @return 트랜잭션 코드
     */
    public String getTransactionCode(String message) {
        return StringUtils.substring(message, 0, 10);
    }
}

File: src/main/kotlin/org/antop/kotlin/chapter2/ImportFunction.kt
package org.antop.kotlin.chapter2

import org.apache.commons.lang3.StringUtils.substring

class ImportFunction {
    /**
     * 전문에서 트랜잭션 코드 추출
     *
     * @param message 전문
     * @return 트랜잭션 코드
     */
    fun getTransactionCode(message: String): String {
        return substring(message, 0, 10)
    }
}
